package.json

```diff
@@ -40,6 +41,7 @@
     "font-awesome": "^4.7.0",
     "font-subset-loader2": "^1.1.7",
     "ga-gtag": "^1.0.1",
+    "gtag": "^1.0.1",
```

Comment:
We probably don't want to include this dependency (https://www.npmjs.com/package/gtag)